src/AppMenuSearch.cc

```
@@ -0,0 +1,413 @@
+void AppMenuSearch::rebuildSearchCandidatesIfNeeded()
+{
+    if (!m_searchCandidatesDirty) {
+        return;
+    }
+    m_searchCandidatesDirty = false;
+    m_searchCandidates.clear();
+
+    if (!m_appMenuModel) {
+        return;
+    }
+    QMenu *rootMenu = m_appMenuModel->menu();
```

Clang (14.0.6)

[warning] variable 'rootMenu' is not initialized

(cppcoreguidelines-init-variables)